### DISCUSSION TITLE
Integration with an application

### DISCUSSION THREAD

**Discussion by PaulCardoos** [bluenviron/mediamtx#452] [Questions]
Hi thank you for this,  it is an awesome server. I was wondering if there is an easy way to integrate this with an application. for example i want to be able to push a stream to it and see active streams. Any ideas?

**Comment by dorinclisu** [comment]
It is easy to integrate, you just need to edit the configuration yaml and the server will automatically load the changes.

**Comment by aler9** [comment]
An HTTP API is available starting from 0.17.0 - you can use it to interact with the server.

**Reply by PaulCardoos** [reply]
Just checked it out. Looks awesome!

[a comment by an automated account is not shown]
